### ISSUE TITLE
Steam User List Pagination

### ISSUE THREAD

**Issue by Sh4kE** [Autostew/autostew#114]

**Comment by jardiacaj** [comment]
Searching and sorting is now broken :(

**Comment by Sh4kE** [comment]
Yep I know, it's unfortunate. However the loading of players took so long in the meantime that we could not have all players loaded anymore. 

I tried to search for pagination frameworks, which support sorting as well. I tried a couple, but they did not work properly or couldn't be compiled with python3 :/ 
I guess we have to implement the sorting ourself by making a new request with a search parameter when clicking on the table headers.
